server/src/main/resources/com/broadcom/lsp/cobol/core/strategy/SpecialTokenHandling.properties

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2020 Broadcom.
+# The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#    Broadcom, Inc. - initial API and implementation
+#
+#
+
```

Comment:
This file presents mapping between token names and what we want to display. For literal token, by default, literal would be displayed in the message but for the rules, rule name would be displayed. In case we want to use a different message/name instead of rule name, we can use this file.

Reply:
Will it make sense if we add also the internal tokens here mapped to an empty value?

Reply:
make sense to do it. Will push in a small change for this.